README.rst

```diff
@@ -13,78 +14,44 @@ Install and update using `pip`_:
 
 .. code-block:: text
 
-  $ pip install -U flask-sqlalchemy
-
-We encourage you to use a virtualenv_. Check the docs for complete installation, version
-requirement, and usage instructions.
+  $ pip install -U Flask-SQLAlchemy
 
 
 A Simple Example
 ----------------
 
-Pretty easy to get started:
-
 .. code-block:: python
 
-  from flask import Flask
-  from flask_sqlalchemy import SQLAlchemy
-
-  app = Flask(__name__)
-  app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:////tmp/test.db'
-  db = SQLAlchemy(app)
-
-
-  class User(db.Model):
-      id = db.Column(db.Integer, primary_key=True)
-      username = db.Column(db.String(80), unique=True, nullable=False)
-      email = db.Column(db.String(120), unique=True, nullable=False)
-
-For more details, see the `quickstart example
-<http://flask-sqlalchemy.pocoo.org/latest/quickstart/#a-minimal-application>`_.
+    from flask import Flask
+    from flask_sqlalchemy import SQLAlchemy
 
+    app = Flask(__name__)
+    app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///example.sqlite"
+    db = SQLAlchemy(app)
 
-Tests
------
-
-In short we use pytest_ and tox_.  You should be able to::
-
-  $ git clone git@github.com:pallets/flask-sqlalchemy.git
-  $ cd flask-sqlalchemy
-  $ tox
-
-That will run everything, so you  might want to start with something like `tox -e py37` or whatever
-version of Python you have installed.
 
-You can figure out our testing and documentation dependencies by further inspecting the `tox.ini`
-file in the root of the project.
```

Comment:
I think it's worth keeping this section as it makes it easier for potential contributors to know how to get started.  Or, if not here, then maybe add it to the docs.

Reply:
I'm working on standard contributing docs for Pallets.